Doc/howto/enum.rst

```diff
@@ -1050,10 +1050,14 @@ If you give your enum subclass extra methods, like the `Planet`_
 class below, those methods will show up in a :func:`dir` of the member,
 but not of the class::
 
-    >>> dir(Planet)                         # doctest: +SKIP
-    ['EARTH', 'JUPITER', 'MARS', 'MERCURY', 'NEPTUNE', 'SATURN', 'URANUS', 'VENUS', '__class__', '__doc__', '__members__', '__module__']
-    >>> dir(Planet.EARTH)                   # doctest: +SKIP
```

Comment:
I assume removing the doctest flag is intentional?

Reply:
No reason to leave it in now that the output is correct, right?

Reply:
Ah, `Planet` isn't defined until later in the file. Good catch! Reverted the commit.